plugins/clipboard/plugin.js

```diff
@@ -502,6 +569,32 @@
 					};
 				} );
 			}
+
+			// Detect if any of paste buttons was touched. In such case we assume that user is using
+			// touch device and force displaying paste dialog (#595).
+			if ( editor.ui.addButton ) {
+				// Waiting for editor instance to be ready seems to be the most reliable way to
+				// be sure that paste buttons are already created.
+				editor.once( 'instanceReady', function() {
+					if ( !editor._.pasteButtons ) {
+						return;
+					}
+
+					CKEDITOR.tools.array.forEach( editor._.pasteButtons, function( name ) {
+						var pasteButton = editor.ui.get( name );
+						// Check if button was not removed by `removeButtons` config.
+						if ( pasteButton ) {
+							var buttonElement = CKEDITOR.document.getById( pasteButton._.id );
+
+							buttonElement.on( 'touchend', function() {
+								editor._.forcePasteDialog = true;
+							} );
+						}
+					} );
+
+					delete editor._.pasteButtons;
```

Comment:
Let's remove this member in the editor, so it could be reused by other plugins if needed this workarounds for mobile.